### PR TITLE
docs(grid): Added docs for column moving #935

### PR DESCRIPTION
**Pull request by sboykova**
Related to IgniteUI/igniteui-angular#935

### CODE REVIEW

File: en/components/grid_column_moving.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+<div class="divider--half"></div>
+
+#### Outputs
+The following `IgxGridComponent` outputs are available for **column moving**:
+
+| Name | Type | Description |
+| :--- | :--- | :--- |
+|`onColumnMoving`| EventEmitter |Emitted throughout the column moving operation. Returns the source and target columns objects. This event is cancelable.|
+|`onColumnMovingEnd`| EventEmitter |Emitted when a column moving ends. Returns the source and target columns objects. This event is cancelable.|
```
Comment (ig-ealbert) [inline en/components/grid_column_moving.md]: I'd suggest either "Emitted when column moving ends." or "Emitted when a column moving operation ends."

File: en/components/grid_column_moving.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+|`onColumnMovingStart`| EventEmitter |Emitted when a column moving starts. Returns the moved column object.|
```
Comment (ig-ealbert) [inline en/components/grid_column_moving.md]: Same here.

File: en/components/grid_column_moving.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
+---
+title: Column Moving - Native Angular | Ignite UI for Angular
+_description: Column moving feature provides means for reordering columns interactively via a standard drag/drop mouse gesture.
```
Comment (ig-ealbert) [inline en/components/grid_column_moving.md]: "provides a means" or "provides a way to reorder columns"

File: en/components/grid_column_moving.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+#### Overview
+**Column moving** feature is enabled on a per-column level, meaning that the **igx-grid** can have a mix of movable and immovable columns. This is done via the `movable` input of the `igx-column`.
+
+```html
+<igx-column [field]="'Category'" [movable]="true"></igx-column>
+```
+
+#### Events
+There are several events related to the column moving to provide means of tapping into the columns' drag and drop operations. These are `onColumnMovingStart`, `onColumnMoving` and `onColumnMovingEnd`.
```
Comment (ig-ealbert) [inline en/components/grid_column_moving.md]: Same feedback on "means" here.

File: jp/components/grid_column_moving.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
+﻿---
+title: 列移動 - ネイティブ Angular | Ignite UI for Angular
+_description: Column moving feature provides means for reordering columns interactively via a standard drag/drop mouse gesture.
```
Comment (ig-ealbert) [inline jp/components/grid_column_moving.md]: @yoshimis This should be translated.

File: jp/components/grid_column_moving.md [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+_keywords: Ignite UI for Angular, UI コントロール, Angular ウィジェット, web ウィジェット, UI ウィジェット, Angular, ネイティブ Angular コンポーネント スイート, ネイティブ Angular コントロール, ネイティブ Angular コンポーネント ライブラリ, Angular Data Grid コンポーネント, Angular Data Grid コントロール, ネイティブ Angular コンポーネント, Angular Grid コンポーネント, Angular Grid コントロール, Angular 高いパフォーマンス Grid, 列のサイズ変更, 遅延列サイズ変更, グリッド列のサイズ変更, Angular Grid 列のサイズ変更, Angular 列
+_language: ja
+---
+
+### グリッドの列移動機能
+
+Ignite UI for Angular の Grid コンポーネントは、標準ドラッグ/ドロップのマウス ジェスチャで **column moving** を使用して列の並び替えができます。
```
Comment (ig-ealbert) [inline jp/components/grid_column_moving.md]: "column moving" should be 列移動 as it is a feature name, not a control name.

File: jp/components/grid_column_moving.md [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+    <iframe id="column-moving-sample-iframe" src='{environment:demosBaseUrl}/grid-moving-sample' width="100%" height="100%" seamless frameBorder="0" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+<br/>
+<div>
+<button data-localize="stackblitz" class="stackblitz-btn" data-iframe-id="column-moving-sample-iframe" data-demos-base-url="{environment:demosBaseUrl}">StackBlitz で開く</button>
+</div>
+<div class="divider--half"></div>
+
+#### 概要
+**列移動は各列レベルで有効にできます。つまり、i**igx-grid** に移動可能な列と移動不可の列の両方を含むことができます。`igx-column` の `movable` 入力によって制御されます。
```
Comment (ig-ealbert) [inline jp/components/grid_column_moving.md]: Stray "i" here should be removed.

Comment (ig-ealbert) [inline jp/components/grid_column_moving.md]: Also need to end the bold for "column moving" (**列移動**)

File: jp/components/grid_column_moving.md [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+
+```html
+<igx-column [field]="'Category'" [movable]="true"></igx-column>
+```
+
+#### イベント
+列のドラッグアンドドロップ操作をカスタマイズするための列移動に関連するイベントが複数あります。`onColumnMovingStart`、`onColumnMoving`、`onColumnMovingEnd` があります。
+igx-grid の `onColumnMovingEnd イベントを処理し、列が新しい位置にドロップされたときにカスタム ロジックを実装できます。たとえば、Change On Year(%) 列の後に Category のドロップをキャンセルできます。
```
Comment (ig-ealbert) [inline jp/components/grid_column_moving.md]: `onColumnMovingEnd` needs to have the ending backtick.

File: jp/components/grid_column_moving.md [hunk 9]
```diff
@@ -0,0 +1,90 @@
...
+    if (event.source.field === "Category" && event.target.field === "Change On Year(%)") {
+        event.cancel = true;
+    }
+}
+```
+
+### API まとめ
+
+#### 入力
+以下の `IgxColumnComponent` 入力は、**column moving** の構成に使用できます。
```
Comment (ig-ealbert) [inline jp/components/grid_column_moving.md]: Please translate feature names.

File: jp/components/grid_column_moving.md [hunk 10]
```diff
@@ -0,0 +1,90 @@
...
+
+| 名前 | タイプ | 説明 |
```
Comment (ig-ealbert) [inline jp/components/grid_column_moving.md]: We've been using 型 for type.

File: jp/components/grid_column_moving.md [hunk 11]
```diff
@@ -0,0 +1,90 @@
...
+| :--- | :--- | :--- |
+|`movable`|boolean|列を移動可能に設定します。|
+
+<div class="divider--half"></div>
+
+#### 出力
+以下の `IgxGridComponent` 出力は **column moving** で使用できます。
```
Comment (ig-ealbert) [inline jp/components/grid_column_moving.md]: Please translate feature names.

File: jp/components/grid_column_moving.md [hunk 12]
```diff
@@ -0,0 +1,90 @@
...
+
+| 名前 | タイプ | 説明 |
+| :--- | :--- | :--- |
+|`onColumnMoving`| 列が移動されたときに発生されます。ソースとターゲットの列オブジェクトを返します。This event is cancelable.|
```
Comment (ig-ealbert) [inline jp/components/grid_column_moving.md]: このイベントはキャンセルできます。

File: jp/components/grid_column_moving.md [hunk 13]
```diff
@@ -0,0 +1,90 @@
...
+| 名前 | タイプ | 説明 |
+| :--- | :--- | :--- |
+|`movable`|boolean|列を移動可能に設定します。|
+
+<div class="divider--half"></div>
+
+#### 出力
+以下の `IgxGridComponent` 出力は **column moving** で使用できます。
+
+| 名前 | タイプ | 説明 |
```
Comment (ig-ealbert) [inline jp/components/grid_column_moving.md]: 型

**Comment by ig-ealbert** [comment]
@yoshimis Could you please make updates here based on Svetla's changes and my review?

File: jp/components/grid_column_moving.md (hunk 7 above)
Comment (ig-ealbert) [reply jp/components/grid_column_moving.md]: @yoshimis This is not fixed.

Comment (ig-ealbert) [reply jp/components/grid_column_moving.md]: @yoshimis This is not fixed.

File: jp/components/grid_column_moving.md (hunk 8 above)
Comment (ig-ealbert) [reply jp/components/grid_column_moving.md]: @yoshimis This is not fixed.

**Comment by sboykova** [comment]
@ig-ealbert This is not urgent.

**Pull request merged**